cpp/include/cudf/detail/row_operator/hashing.cuh

```diff
@@ -134,29 +138,30 @@ class device_row_hasher {
    */
   template <template <typename> class hash_fn>
```

Comment:
Optional: If you want to rename `hash_fn` to `hash_function`, that'd increase consistency a bit.